src/main/java/seedu/address/model/client/Client.java

```diff
@@ -54,12 +56,8 @@ public Address getAddress() {
         return address;
     }
 
-    public Meeting getMeeting() {
-        return meeting;
-    }
-
-    public void setMeeting(Meeting meeting) {
-        this.meeting = meeting;
+    public ArrayList<Meeting> getMeetings() {
```

Comment:
Perhaps rather than exposing the ArrayList, we can return some sort of return only list since the use of getMeetings should only be to show the meeting details on the app.